app/views/application/_navbar.html.erb

```diff
@@ -15,6 +15,9 @@
       <li class="nav-item">
         <a class="nav-link" href="../servers">Servers</a>
       </li>
+      <li class="nav-item">
+        <%= link_to('Users', users_path, class: "nav-link") %>
```

Comment:
#77 does need a button to access the users profile, is this not the path for the index of users?

Reply:
I see for #84 its the users and the profile is still missing =)

Reply:
I am not quite sure, what is missing?

Reply:
We should have a button to our own profile page? Maybe display first and last name in the menu bar and when you press it you get redirected to your own profile?

Reply:
Done